src/scripts/main.js

```diff
@@ -1,5 +1,45 @@
+/* eslint-disable no-undef */
+// they sad "getComputedStyle is not undefined"
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+const spider = document.querySelector('.spider');
+const spiderSide = parseFloat(getComputedStyle(spider).width);
```

Comment:
it's better to use spider.clientWidth or spider.offsetWidth instead of CSS width